plugins/ccgx/fu-ccgx-cyacd-file.c

```diff
@@ -0,0 +1,458 @@
+/*
+ * Copyright (C) 2020 Cypress Semiconductor Corporation.
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include <string.h>
+#include "fu-ccgx-cyacd-file.h"
+
+static guint8
+ch_to_hex (guint8 value)
```

Comment:
Nak. This poorly reimplements g_ascii_strtoull() and I think you should be using fu_firmware_strparse_uint16() anyway.

Reply:
I will replace ch_to_hex with fu_firmware_strparse_uint8.

Reply:
I fixed it.